src/components/package/PackageCard.tsx

```diff
@@ -83,9 +83,17 @@ const PackageCard: React.VFC<
   programDateEnabled: datetimeEnabled,
   deliveredAt,
 }) => {
+  const location = useLocation()
+  const pathname = location.pathname
   return (
     <>
-      <Link to={isExpired ? `/program-packages/${id}` : `/program-packages/${id}/contents?memberId=${memberId}`}>
+      <Link
+        to={
+          !isExpired && pathname.includes('members')
+            ? `/program-packages/${id}/contents?memberId=${memberId}`
+            : `/program-packages/${id}`
+        }
+      >
```

Comment:
I think moving the logic for determining the link out of the <PackageCard> component is better, as the hard-coded path judgment is not flexible.

Reply:
for example:
```
// MemberPage
const toLink = isExpired
    ? `/program-packages/${programPackage.id}`
    : `/program-packages/${programPackage.id}/contents?memberId=${memberId}`;

// Others
const toLink =`/program-packages/${programPackage.id}`
    
<PackageCard
    to={toLink}
    ......
   />
   
```

Reply:
OK, adjusted, thanks for reviewing